lib/marc_links.rb

```diff
@@ -37,7 +37,7 @@ def as_h
         link_title:,
         additional_text:,
         material_type: field['3'],
-        note: field['z'],
+        note: subzs,
```

Comment:
I don't have a problem with this but I am wondering why the finding_aid? field isn't using this?

Reply:
I'm not sure (and one of the reasons I merged #1579 as is). It seems pretty funny that we'd only look at the first `$z` 🤷‍♂️